src/Eshopworld.Messaging/Eshopworld.Messaging.csproj

```diff
@@ -2,8 +2,8 @@
 
   <PropertyGroup>
     
-    <Version>1.0.1</Version>
-    <PackageReleaseNotes>Added an internal optional route to bypass event deserialization during subscribe.</PackageReleaseNotes>
+    <Version>1.1.0-beta4</Version>
```

Comment:
Shouldn't the major version be bumped up (because of eshopworld.core)?

Reply:
Depends on the point of view:
- There's no breaking change here.
- All packages that depend on Core (which are all, except tests-core), will need a new release to align with Core, bumping major on all of them is misleading as having significant feature improvements.

Reply:
Potentially it's a breaking change. Everything that requires 2.0 will be silently redirected to 3.0. Depending on changes made between these versions both build and runtime errors are possible. IMHO (and according to rules of semantic versioning) major version has nothing to do with "significant feature improvements" but with compatibility. The former frequently causes the later, but incompatibility can happen without big changes.